takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/classpath/DependencyClasspathEntry.java

```diff
@@ -121,6 +122,11 @@ public String getEntryDescription() {
     return sb.toString();
   }
 
+  @Override
+  public Path getLocation() {
+    return file.toPath();
```

Comment:
What branch are you using? [`#file` has type `Path` in the current master](https://github.com/takari/takari-lifecycle/blob/cb57d0eaeee89e8c74a2ccf29a64fa0bfe593be4/takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/classpath/DependencyClasspathEntry.java#L37).

Reply:
My fork is 5 commits behind master, so I'll have to rebase the branch.